draft-pauly-httpbis-geoip-hint.md

```diff
@@ -27,33 +27,56 @@ author:
 
 --- abstract
 
-This documents defines an HTTP Client Hint that allows a client to share information
-about its IP Geolocation. This helps ensure that servers have information about location
-that is consistent with what a client expects and what other servers use.
+This document defines an HTTP Client Hint that allows the client to
+share its location, reducing the need for destination servers to map IP
+ranges to locations. By actively providing their location, clients gain
+greater influence and transparency regarding their geolocation.
+Additionally, when IP-hiding technologies like VPNs or proxies are
+used, servers can still deliver geographically relevant experiences
+without relying on large IP address pools to cover regional areas or
+costly third-party GeoFeeds, which can be outdated and inaccurate at
+times.
 
 --- middle
 
 # Introduction {#introduction}
 
-HTTP Client Hints {{!RFC8942}} defines a convention for HTTP headers
-to communicate optional information from clients to servers as hints. This can be done
-conditionally based on if a server claims supports for a particular hint.
-
-This document defines a client hint that can be used to send a IP geolocation entry that
-maps to the client's IP address. This location can be used to influence server behavior,
-such as by causing the server to return responses relevant to the client's location.
-The format of the IP geolocation entry is the same as that defined for IP geolocation
-feeds in {{!GEOFEED=RFC8805}}.
-
-This header is intended to be used to provide rough geolocation hints to servers that do
-not already have accurate or authoritative mappings for the IP addresses of clients. This
-can be particularly useful for cases where IP geolocation mappings have changed recently,
-or a client is using a VPN or proxy that may not be commonly recognized by servers.
-
-The mechanism for how a client learns the IP geolocation mapping to send is beyond the
-scope of this document. {{?RFC9092}} defines some mechanisms for discovery, but clients
-can also have other mechanisms (such as coordinating with a VPN or proxy that is assigning
-the client a tunnelled or proxied address) to learn what hint to sent.
+HTTP Client Hints {{!RFC8942}} defines a convention for HTTP headers to
+communicate optional information from clients to servers as hints. This
+can be done conditionally based on if a server claims to support a
+particular hint. A server requests hints by listing them in the
+Accept-CH response header, or via other methods such as ALPS or
```

Comment:
If you include ALPS, it would need a reference. But ALPS is not something that's actually adopted anywhere right now from what I know, so I wouldn't reference it at all.

Reply:
+1, I would just skip ALPS, mainly because ALPS failed at IETF and we don't want to be associated with that

Reply:
Got it removed!